frontend/src/metabase/lib/i18n.js

```diff
@@ -74,10 +74,30 @@ export function setLocalization(translationsObject) {
   // eslint-disable-next-line react-hooks/rules-of-hooks
   useLocale(locale);
 
-  moment.locale(locale);
+  updateMomentLocale(locale);
   updateMomentStartOfWeek(locale);
 }
 
+function updateMomentLocale(locale) {
+  const momentLocale = mapToMomentLocale(locale);
+  if (momentLocale !== "en") {
+    require("moment/locale/" + momentLocale);
+  }
+
+  moment.locale(momentLocale);
+}
+
+function mapToMomentLocale(locale) {
+  switch (locale) {
+    case "zh-Hans":
+      return "zh-cn";
+    case "zh-TW":
+      return "zh-tw";
+    default:
+      return locale;
```

Comment:
Couldn't the default just be to lowercase, since that seems like the Moment convention?

Reply:
Yeah, good idea.